server/utils/response.go

```diff
@@ -7,21 +7,22 @@ import (
 )
 
 type errorResp struct {
-	Success   bool      `json:"success"`
-	Message   string    `json:"message"`
-	TimeStamp time.Time `json:"time_stamp"`
+	Success   bool   `json:"success"`
+	Message   string `json:"message"`
+	TimeStamp string `json:"time_stamp"`
```

Comment:
For timestamp, it is recommended not to send a formatted date time string as different location has its way of showing formatted date time string. It is better send the number of milliseconds elapsed since January 1, 1970 00:00:00 UTC instead.

Reply:
Actually this issue is the same for all time stamp fields, forgot to mention it in the previous PRs.